### ISSUE TITLE
Installer not working in hello-0.5.0_0d10343e6301ba30721e4371cc088d460c8db1ad-FreeBSD-12.2-amd64

### ISSUE THREAD

**Issue by grahamperrin** [helloSystem/hello#158]
`hello-0.5.0_0d10343e6301ba30721e4371cc088d460c8db1ad-FreeBSD-12.2-amd64.iso`

The installer disappears after clicking **Continue** at the license page.

**Comment by grahamperrin** [comment]
https://github.com/helloSystem/ISO/issues/167 was different; IIRC the installer closed/disappeared after the **First** Continue button.

**Comment by dbisping** [comment]
Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe Unsubscribe 

O the joy of my spirit - it is uncaged - it darts like lightning!
It is not enough to have this globe or a certain time,
I will have thousands of globes and all time.

- Walt Whitman

**Comment by grahamperrin** [comment]
… or maybe close this as a duplicate? 

That is, if there's _generally_ no expectation of installations working until after _other_ things are made to work reliably with [unionfs(8)](https://www.freebsd.org/cgi/man.cgi?unionfs(8)) …

**Comment by probonopd** [comment]
Indeed, there's generally no expectation of installations working on unionfs based ISOs until https://github.com/helloSystem/ISO/issues/4#issuecomment-801349277 is closed.
